src/routes/trading-view/[chain]/[exchange]/export-data/+page.svelte

```diff
@@ -73,39 +73,39 @@
 
 			<div>
 				<label for="export_format">Export format</label>
-				<select class="form-control" id="export_format" bind:value={selectedFormat}>
+				<Select class="form-control" id="export_format" bind:value={selectedFormat}>
 					<option value="excel">Microsoft Excel .xlsx</option>
```

Comment:
Somewhat unrelated to this PR… but since we're touching this code…

I wonder if we should make this `disabled` since there's currently only one option.

Reply:
I think that adding `disabled` state might actually be misleading... Users might think that because it's disabled it does not matter or is not considered important.